master/getting-started/docker/installation/manual.md

```diff
@@ -1,101 +1,68 @@
-on two Linux servers that have a number of installation requirements.
+Calico's core components are shipped in a Docker container.
+Calico installation requires this container run on each host.
+Calico also provides a command line tool, `calicoctl`, which simplifies launching
+this container.
 
-This tutorial describes how to manually configure a working environment for
-the example.
+## Using Calicoctl
 
-## Requirements
+1. Download the calicoctl binary:
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+	 ```
+   sudo wget -O /usr/local/bin/calicoctl http://www.projectcalico.org/builds/calicoctl
+	 sudo chmod +x calicoctl
+   ```
 
-We recommend configuring the hosts with the hostname `calico-01` and
-`calico-02`.  The worked example will refer to these hostnames.
+2. Launch `calico/node`:
 
-They must have the following software installed:
+   ```
+   sudo calicoctl node run
+   ```
 
-- Docker 1.9 or greater (details below)
-- `ip_set`, `iptables`, and `ip6tables` kernel modules.
--  The `calicoctl` binary in your path (see below)
+The `node run` command starts Calico's core services in the `calico/node`
+docker container. Check that they are running:
 
-You will also need an etcd cluster which Calico uses for coordinating state
-between the nodes.  This may installed on one or both of the two servers for
-the worked example.  See the [etcd documentation][etcd] for details on setting
-up a cluster.
+```
+vagrant@calico-01:~$ docker ps
+CONTAINER ID        IMAGE                COMMAND             CREATED             STATUS              PORTS               NAMES
+408bd2b9ba53        calico/node:v1.0.0-rc2   "start_runit"       About an hour ago   Up About an hour                        calico-node
+```
 
-### Docker (with multi-host networking)
+Furthermore, check that the `calico/node` container is functioning properl
+with the following command:
 
-Follow the instructions for installing [Docker][docker].  A version of 1.9 or
-greater is required.
+```
+sudo calicoctl node checksystem
+```
 
-To use Calico as a Docker network plugin, the Docker daemon
-needs to run specifying a cluster store.  If using etcd as a cluster store,
-configure the `cluster-store` on the Docker daemon to `etcd://<ETCD_IP>:<ETCD_PORT>`,
-replacing `<ETCD IP>` and <ETCD_PORT> with the appropriate address and client
-port for your etcd cluster.
+## Using "docker run"
 
-> For Docker 1.10+, you can use the [daemon configuration file][daemon-config-file],
-> or for 1.9 see the appropriate 'Configuring Docker' section in [configuring docker][configuring-docker-1.9].
+For more control over the Calico startup process, and to simplify binding
+startup to an init system, `calicoctl` can print the command it uses
+to launch `calico/node`.
 
-#### Docker permissions
+To print the command `calicoctl node run` uses to launch Calico on this host,
```

Comment:
You want to run with the `--dry-run` option so that it doesn' actually run the command.

Is the text below from a recent version of the tool - I think it shouldn't have the last para and it should also include the command to use to stop the container.